dashboard/README.md

```
@@ -0,0 +1,106 @@
+make api        # Terminal 1
+make dashboard  # Terminal 2
+```
+
+## Docker
+
+```bash
+# From project root
+make docker-up
+```
+
+Dashboard available at http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)